src/app/model/tollbooth.ts

```diff
@@ -56,4 +57,9 @@ export class Tollbooth {
     return this.locktime - Math.log(1 - Math.random()) * this.rate;
   }
 
+  private render(): void {
```

Comment:
The ref produced here should be saved somewhere in the tollbooth somehow, or produced at creation time and then only updated here